### PR TITLE
chore(message-system): refresh uuids for soft and hard suite update b…

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Bump soft and hard warning for Suite update + cleanup of unused message. 

Tracked in Notion [here](https://www.notion.so/refresh-uuids-for-soft-and-hard-suite-update-banners-150dc5260606803dbc6cc85d7522a4ac?pvs=4)

### CODE REVIEW

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -56,97 +56,6 @@
...
-                }
-            ],
-            "message": {
-                "id": "b821e8f6-ea24-4bc9-9c28-e603a98cbec3",
-                "priority": 93,
-                "dismissible": false,
-                "variant": "info",
-                "category": ["context"],
-                "content": {
-                    "en-GB": "For the best staking experience, please update to the latest version of Trezor Suite.",
```
Comment (MiroslavProchazka) [inline suite-common/message-system/config/config.v1.json:76]: We do not need this message anymore, thus deleting.

File: suite-common/message-system/config/config.v1.json [hunk 2]
```diff
@@ -387,16 +296,16 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.10.1",
+                        "desktop": "<=24.5.4",
```
Comment (MiroslavProchazka) [inline suite-common/message-system/config/config.v1.json:299]: 24.5.4 is the Intermediary version signed by new certificate. This version looks to our servers for new updates, not Github. It is time to migrate majority of users to at least this version so we can kill Github releases in the long run eventually.

File: suite-common/message-system/config/config.v1.json [hunk 3]
```diff
@@ -439,14 +348,14 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<=24.6.3",
+                        "desktop": "<=24.10.1",
```
Comment (MiroslavProchazka) [inline suite-common/message-system/config/config.v1.json:351]: This is kind of drastic as it is only 1 month old Suite. The reasoning behind is that the 24.10.1 contains bug in send form that is already fixed in 24.11.3 and we want users to migrate quickly,

**Review by tomasklim — APPROVED** [review]

**Comment by bosomt** [comment]
**24.10.1**

image

**24.5.4**
image

**Pull request merged**